templates/block.hbs

```diff
@@ -11,7 +11,11 @@
       </span>
     </div>
     <div class="results">
-      {{{entityValue}}}
+      {{#if block.entity.containsHtml}}
+        {{entityValue}}
+      {{else}}
+        {{{entityValue}}}
```

Comment:
@penwoodjon Is there a reason why we want to display unescaped output here?

Reply:
@sarus The reason being is if there is links or other things that can run using html or script tags in the entity value.  There are times in which you want to directly interact with the html text as well, so this is for those cases as well.